forc-pkg/tests/invalid/Forc.toml

```diff
@@ -0,0 +1,3 @@
+[dependencies]
```

Comment:
nit: consider putting this under `forc-pkg/tests/invalid/duplicate_keys/Forc.toml` in case we add more tests of invalid Forc manifests (which we should!)

Reply:
I agree with you @sdankel. Maybe a feature is missing or upgrading. I will try to get the same error upgrading the library. I'll keep you posted.